drivers/clock_control/clock_control_wch_rcc.c

```diff
@@ -154,6 +154,7 @@ static int clock_control_wch_rcc_init(const struct device *dev)
 			RCC->CFGR0 |= RCC_PLLSRC;
```

Comment:
A suggestion for the subject line: "drivers: clock_control: disable the HSI PLL prescaler"

That describes what the change does, and the body can describe why such as "For consistency with other SoCs in the family, ..."

Note that this PR does change behaviour. Do a quick grep of the current boards and see if anyone is using the hsi.